packages/storefront-theme/style.css

```
@@ -606,6 +855,586 @@ textarea:focus-visible {
+/* -------------------------------------------------------------------------
+ * Fresh Dispatch — selected 2026-08-27 visual direction.
+ * The aubergine field, numbered aisles, compressed display type, and gapless
+ * product board are the canonical Modern Grocery signature.
+ * ---------------------------------------------------------------------- */
+
+:root {
+	--grovia-dispatch-ink: #1e1424;
+	--grovia-dispatch-tomato: #ff4b2b;
+	--grovia-dispatch-paper: #fffdf8;
+	--grovia-dispatch-apricot: #f6d8c8;
+	--grovia-display: Impact, Haettenschweiler, "Arial Narrow Bold", "Arial Narrow", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Impact" to be "impact" (value-keyword-case)

(value-keyword-case)